pkg/services/ngalert/notifier/channels/discord.go

```diff
@@ -34,6 +35,8 @@ func NewDiscordNotifier(model *NotificationChannelConfig, t *template.Template)
 		return nil, alerting.ValidationError{Reason: "No Settings Supplied"}
 	}
 
+	avatarURL := model.Settings.Get("avatar_url").MustString()
```

Comment:
For the ngalert, you will need to add this option [here](https://github.com/grafana/grafana/blob/91657dad182127bf577b449ff9d94e5bf86e592a/pkg/services/ngalert/notifier/available_channels.go#L669-L692
) and updated [this test](https://github.com/grafana/grafana/blob/91657dad182127bf577b449ff9d94e5bf86e592a/pkg/tests/api/alerting/api_available_channel_test.go) accordingly.